src/main/java/com/GeorgesServer/app/RequestParser.java

```diff
@@ -24,10 +25,12 @@ public ClientRequest parse(InputStream reader) {
         String request = requestReader.read(reader);
         parseRequestStartLine(request);
         parseHeaders(request);
+        parseBody(request);
```

Comment:
I *do not* want you to make the change I'm about to talk about, it's just something to think about in the future. When you're destructively operating on a piece of data, it increase clarity to make it really clear what changes are happening at the top level. Super simple example: instead of 

```
top_level_method(data):
    addGizmo(data)
    ...
```
you can have 
```
top_level_method(data):
    data['gizmo'] = this.get_gizmo()
    ...
```

It's not possible everywhere, but it can save readers from rummaging through a whole file to understand the ways data is being changed!